datasets/conv_ai_3/README.md

```diff
@@ -0,0 +1,190 @@
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/aliannejadi/ClariQ
+- **Repository:** https://github.com/aliannejadi/ClariQ
+- **Paper:** https://arxiv.org/abs/2009.11352
+- **Leaderboard:** [More Information Needed]
+- **Point of Contact:** [More Information Needed]
+
+### Dataset Summary
+
+The Conv AI 3 challenge is organized as part of the Search-oriented Conversational AI (SCAI) EMNLP workshop in 2020. The main aim of the conversational systems is to return an appropriate answer in response to the user requests. However, some user requests might be ambiguous. In Information Retrieval (IR) settings such a situation is handled mainly through the diversification of search result page. It is however much more challenging in dialogue settings. Hence, we aim to study the following situation for dialogue settings:
+
+- a user is asking an ambiguous question (where ambiguous question is a question to which one can return > 1 possible answers)
+- the system must identify that the question is ambiguous, and, instead of trying to answer it directly, ask a good clarifying question.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+```
+{2287: {'topic_id': 8,
+'facet_id': 'F0968',
+'initial_request': 'I want to know about appraisals.',
+'clarification_need': 2,
+'question_id': 'F0001',
+'question': 'are you looking for a type of appraiser',
+'answer': 'im looking for nearby companies that do home appraisals',
+'facet_desc': 'Get the TYPE of Appraisals'
+'conversation_context': [],
+'context_id': 968},
+2288: {'topic_id': 8,
+'facet_id': 'F0969',
+'initial_request': 'I want to know about appraisals.',
+'clarification_need': 2,
+'question_id': 'F0005',
+'question': 'are you looking for a type of appraiser',
+'facet_desc': 'Get the TYPE of Appraisals'
+'answer': 'yes jewelry',
+'conversation_context': [],
+'context_id': 969},
+1570812: {'topic_id': 293,
+'facet_id': 'F0729',
+'initial_request': 'Tell me about the educational advantages of social networking sites.',
+'clarification_need': 2,
+'question_id': 'F0009'
+'question': 'which social networking sites would you like information on',
+'answer': 'i don have a specific one in mind just overall educational benefits to social media sites',
+'facet_desc': 'Detailed information about the Networking Sites.'
+'conversation_context': [{'question': 'what level of schooling are you interested in gaining the advantages to social networking sites',
+'answer': 'all levels'},
+{'question': 'what type of educational advantages are you seeking from social networking',
+'answer': 'i just want to know if there are any'}],
+'context_id': 976573}
```

Comment:
I think it would be more readable this way.
Also it looks like one field is still missing: topic_desc


````suggestion
Here are a few examples from the dataset:
```
{'topic_id': 8,
'facet_id': 'F0968',
'initial_request': 'I want to know about appraisals.',
'clarification_need': 2,
'question_id': 'F0001',
'question': 'are you looking for a type of appraiser',
'answer': 'im looking for nearby companies that do home appraisals',
'facet_desc': 'Get the TYPE of Appraisals'
'conversation_context': [],
'context_id': 968}
```

```
{'topic_id': 8,
'facet_id': 'F0969',
'initial_request': 'I want to know about appraisals.',
'clarification_need': 2,
'question_id': 'F0005',
'question': 'are you looking for a type of appraiser',
'facet_desc': 'Get the TYPE of Appraisals'
'answer': 'yes jewelry',
'conversation_context': [],
'context_id': 969}
```

```
{'topic_id': 293,
'facet_id': 'F0729',
'initial_request': 'Tell me about the educational advantages of social networking sites.',
'clarification_need': 2,
'question_id': 'F0009'
'question': 'which social networking sites would you like information on',
'answer': 'i don have a specific one in mind just overall educational benefits to social media sites',
'facet_desc': 'Detailed information about the Networking Sites.'
'conversation_context': [{'question': 'what level of schooling are you interested in gaining the advantages to social networking sites', 'answer': 'all levels'}, {'question': 'what type of educational advantages are you seeking from social networking', 'answer': 'i just want to know if there are any'}],
'context_id': 976573}
````